packages/backend/src/controllers/collection.controller.ts

```diff
@@ -271,3 +272,52 @@ export async function deleteTradeOffer(req: Request, res: Response) {
   }
 }
 
+// View Comic Book Offers
+
+export async function viewTradeOffers(req: Request, res: Response) {
+  try {
+    const tradeOffers = await viewComicBookOffers();
+
+    if (!tradeOffers.length) {
+      return res
+        .status(404)
+        .json({ error: 'There are currently no trade offers' });
+    }
+
+    return res.status(201).json({
+      status: 'Success',
+      data: {
+        tradeOffers: tradeOffers.map((tradeOffer: any) => ({
+          tradeOfferId: tradeOffer.id,
+          type: tradeOffer.type,
+          status: tradeOffer.status,
+          message: tradeOffer.message,
+          price: tradeOffer.price,
+          createdAt: tradeOffer.createdAt,
+          createdBy: {
+            userId: tradeOffer.createdBy.id,
+            firstName: tradeOffer.createdBy.firstName,
+            lastName: tradeOffer.createdBy.lastName,
+            username: tradeOffer.createdBy.username,
+            profileImage: tradeOffer.createdBy.profileImage,
+            location: tradeOffer.createdBy.location,
+          },
+          contactDetails: {
```

Comment:
Why can't use just get the contact details of the user when are creating an trade offer instead because we still have implemented a way on the frontend to get the user's phone number - and as @fmtabbara mentioned before - we don't to force the user to use the email they signed-in to.

Does that make sense or not?